components/function-controller/hack/test-cluster.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+# easy script to setup
+# TODO add basic trap on err
+
```

Comment:
this script hast to be called from function-controller directory can wee have info about it;
can we have a short info on potential collision with kind in different version, that could be visible via PATH?